### DISCUSSION TITLE
how to upload video

### DISCUSSION THREAD

**Discussion by KalpanaChetan** [community/community#32276] [Other Feature Feedback, Questions, & Ideas] [answered]
![Screenshot_20220908-122128_Iron](https://user-images.githubusercontent.com/112402936/189054486-b8119eb2-1662-49e3-ad02-7a556b4a673a.jpg)
Can't upload video

**Comment by Appfanatic456** [comment]
That's because your uploading an image and not a video. Try uploading a video when pressing the icon.

**Reply by KalpanaChetan** [reply]
Screenshot gets uploaded  but the video is not uploaded


Video size 80 mb

**Reply by KalpanaChetan** [reply]
Maybe the size of the video is too much so it is not uploading

**Comment by KalpanaChetan** [comment]
Can't upload video

**Comment by C3n7ral051nt4g3ncy** [comment]
Hi @KalpanaChetan, 

Simply drag and drop your video into wherever you want to put it.
It has to be a `.Mov` or `.MP4`

I am uploading a video for you right now as a test.

Just make sure the video is the correct format, and you have all the information  [here:](https://github.blog/2021-05-13-video-uploads-available-github/)

If it's not uploading it means it's too heavy and you will need to amend it. 

https://user-images.githubusercontent.com/104733166/189111131-18e2adfb-00c5-435e-94e9-6fe4178315be.mp4

**Reply by KalpanaChetan** [reply]
![Screenshot_20220908-190856_GitHub](https://user-images.githubusercontent.com/112402936/189137048-26cfc039-a81d-4b04-a2f2-f140f4a13a41.jpg)

**Comment by C3n7ral051nt4g3ncy** [comment, marked as answer]
@KalpanaChetan 

What is the size of the video?
It should be no more than `100MB`, if it's more, that's the reason why it can't be uploaded.

**Reply by KalpanaChetan** [reply]
video size 20 MB

**Reply by C3n7ral051nt4g3ncy** [reply]
Do you have a location for this file somewhere online such as YouTube?

**Reply by KalpanaChetan** [reply]
Recorder app  
 az screen  recorder

**Reply by KalpanaChetan** [reply]
> 

No

**Reply by C3n7ral051nt4g3ncy** [reply]
@KalpanaChetan Try converting it to `.Mov ` and try again to upload it:
https://www.freeconvert.com/mp4-to-mov

**Reply by KalpanaChetan** [reply]
Thank you very much for this website 

Problem solve

**Reply by C3n7ral051nt4g3ncy** [reply]
@KalpanaChetan Great news!! , I thought so, often,  recorder apps won't put the right codecs on videos, so the conversion set everything right.
Click on `question answered` so the topic can be closed.

**Reply by KalpanaChetan** [reply]
Ok

**Comment by C3n7ral051nt4g3ncy** [comment]
See the screenshot, it should be green after and will say `Answered` when you confirm your question was answered.

<img width="1122"  src="https://user-images.githubusercontent.com/104733166/189151359-f740cab0-e15a-4859-b2c0-5a1a0c8ed0c7.png">

**Reply by KalpanaChetan** [reply]
Thank again

**Comment by C3n7ral051nt4g3ncy** [comment]
No problem, my pleasure 👍
